src/core/public/ui_settings/ui_settings_service.mock.ts

```diff
@@ -22,7 +22,7 @@ import { IUiSettingsClient } from './types';
 
 const createSetupContractMock = () => {
   const setupContract: jest.Mocked<IUiSettingsClient> = {
-    getAll: jest.fn(),
+    getAll: jest.fn(() => ({})),
```

Comment:
We are splitting the contract declaration and the methods mocked returns to ensure proper typings. (see part 3.1 of `src/core/CORE_CONVENTIONS.md`)

https://github.com/elastic/kibana/blob/0f8533730c4d777669e590f9bc8796f394e55d68/src/core/public/ui_settings/ui_settings_service.mock.ts#L39-L42

Please do the same for `getAll`

Reply:
@pgayvallet I believe this is fixed, would like verification.